CHANGELOG.md

```diff
@@ -1,3 +1,34 @@
+### citus v10.2.4 (February 1, 2022) ###
+
+* Adds support for operator class parameters in indexes
+
+* Fixes a build error that happens when `lz4` is not installed
+
+* Fixes function signature generation bug
+
+* Improves self-deadlock prevention for `CREATE INDEX`, `REINDEX CONCURRENTLY`
+  commands for builds using PG14 or higher
+
+* Uses a fixed `application_name` while connecting to remote nodes
+
+### citus v10.1.4 (February 1, 2022) ###
+
+* Adds missing version checks for columnar tables
+
+* Fixes a bug that could break `DROP SCHEMA/EXTENSION` commands when there is
+  a columnar table
+
+* Fixes a build error that happens when lz4 is not installed
+
+* Fixes a missing `FROM` clause entry error
+
+* Improves self-deadlock prevention for `CREATE INDEX`, `REINDEX CONCURRENTLY`
+  commands for builds using PG14 or higher
+
+* Reinstates optimisation for uniform shard interval ranges
+
+* Uses a fixed `application_name` while connecting to remote nodes
```

Comment:
maybe something like:

Fixes a bug that causes commands to fail when `application_name` is set